### ISSUE TITLE
Litematica Mode after 1.0.5

### ISSUE THREAD

**Issue by PFL-MushroomForge** [beanbag44/Nuker#1]
In 1.0.5 it looks like you removed code related to the litematica mode for only breaking blocks that do not match a litematica schematic. 

Was there a specific reason for removing this code, potentially elsewhere in the project that I just didnt see or fully know how its working?

**Comment by beanbag44** [comment]
hi sorry for the late reply, i believe there is still a setting for litematica which should do that

**Comment by beanbag44** [comment]
yeah just had a look, it checks if the schematicMismatches set contains the block position before breaking it

**Comment by PFL-MushroomForge** [comment]
There is still a litematica setting in the correct versions but in newer releases it doesn't seem to work/function.

**Comment by beanbag44** [comment]
hm, thats odd, ill look into it

**Comment by beanbag44** [comment]
i know its been months (sorry) but the kotlin rewrite has been transferred to this repo finally. There arent any public builds yet as there are a few things that need fixing but if you know how, its in a very usable state if you wanted to build it yourself
